doc/classes/PackedByteArray.xml

```diff
@@ -380,7 +380,10 @@
 			<param index="0" name="at_index" type="int" />
 			<param index="1" name="value" type="int" />
 			<description>
-				Inserts a new element at a given position in the array. The position must be valid, or at the end of the array ([code]idx == size()[/code]).
+				Adds the element ([param value]) at the given index ([param position]) into the array. All elements from that position onward shift one spot to the right to make room, increasing the array's size by one.
```

Comment:
Noticing now that the params are named differently (at_index vs position). Should I rename them to match?

Reply:
I opted to call them all `index`, which is more standard for (godot) documentation.